content/en/docs/concepts/storage/storage-capacity.md

```diff
@@ -0,0 +1,134 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Storage Capacity
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+Storage capacity is limited and may vary depending on the node on
+which a pod runs: network-attached storage might not be accessible by
+all nodes, or storage is local to a node to begin with.
+
+{{< feature-state for_k8s_version="v1.19" state="alpha" >}}
+
+This page describes how Kubernetes keeps track of storage capacity and
+how the scheduler uses that information to schedule Pods onto nodes
+that have access to enough storage capacity for the remaining missing
+volumes. Without storage capacity tracking, it is random whether the
+selected node has enough storage for the Pod and multiple scheduling retries may be
```

Comment:
nit: `it is random`? Could this be reworded.
Without storage capacity tracking, the way a node is selected ...?

Reply:
How about

Without storage capacity tracking, the scheduler may choose a node that doesn't have enough capacity to provision a volume.

Reply:
👍

Reply:
Changed.